src/main/java/com/commercetools/payment/handler/CommercetoolsExecutePaymentsController.java

```diff
@@ -26,13 +30,15 @@ public CommercetoolsExecutePaymentsController(@Nonnull StringTrimmerEditor strin
 
     @RequestMapping(
             method = RequestMethod.POST,
+            consumes = APPLICATION_JSON_VALUE,
+            produces = APPLICATION_JSON_VALUE,
             value = "/{tenantName}/commercetools/execute/payments/")
     public ResponseEntity executePayments(@PathVariable String tenantName,
-                                                 @RequestParam String paypalPlusPaymentId,
-                                                 @RequestParam String paypalPlusPayerId) {
+                                          @Valid @RequestBody PaypalPlusExecuteParams params,
+                                          BindingResult result) {
         PaymentHandleResponse paymentHandleResponse = paymentHandlerProvider
                 .getPaymentHandler(tenantName)
-                .map(paymentHandler -> paymentHandler.executePayment(paypalPlusPaymentId, paypalPlusPayerId))
+                .map(paymentHandler -> paymentHandler.executePayment(params.getPaypalPlusPaymentId(), params.getPaypalPlusPayerId()))
```

Comment:
what happens if validation of parameters fails. Will the caller see nice error response?